web/html/src/manager/content-management/shared/components/messages/messages.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import * as React from 'react';
+import _isEmpty from "lodash/isEmpty";
+
+import {Messages} from 'components/messages';
+import type {ProjectMessageType} from '../../type/project.type';
+
+type ValidationMessagesType = {
+  panelClass: string,
+  messages: ?React.Element<'ul'>
+}
+
+const msgClassMap = {
+  'warning': {
```

Comment:
I think the single quote `'` is not needed for the `Map key` parameter, but only for the value. What about cleaning it up through the whole file? I think we never use quotes for the `key`.